src/amplitude.js

```diff
@@ -190,6 +190,44 @@ class Amplitude {
     // }
   }
 
+  /**
+   *  Disconnects the output of this p5.Amplitude object.
+   *
+   *  @method disconnect
+   *  @for p5.Amplitude
+   *  @example
+   *  <div><code>
+   *  let sound, amplitude;
+   *  function preload(){
+   *    sound = loadSound('assets/beat.mp3');
+   *  }
+   *
+   *  function setup() {
+   *    let cnv = createCanvas(200, 200);
+   *    cnv.mouseClicked(togglePlay);
+   *    amplitude = new p5.Amplitude(); 
+   *    sound.loop();
+   *  }
+   *
+   *  function draw() {
+   *    background(220);
+   *    text('Disconnect:', 20, 20);
+   *    let level = amplitude.getLevel();
+   *    let size = map(level, 0, 1, 0, 400);
+   *    ellipse(width/2, height/2, size, size);
+   *  }
+   *
+   *  function togglePlay() {
+   *    if (sound.isPlaying()){
+   *      sound.pause();
+   *      amplitude.disconnect();
```

Comment:
I think calling `disconnect` here might not be the best illustration of what's going on since `disconnect` only disconnects the amplitude's outputs. If we don't call `sound.pause()` on the line above, nothing appears to happen. Is there an example we can make where we connect the amplitude's output to something else, so that we can disconnect the amplitude's output from that rather than pausing its input?

Reply:
Should we make a separate button for the disconnect method?